src/allotropy/parsers/methodical_mind/methodical_mind_structure.py

```diff
@@ -87,12 +87,14 @@ def create(luminescence: int, location_id: str, well_plate_id: str) -> WellData:
 def create_metadata(header: Header) -> Metadata:
     return Metadata(
         file_name=header.file_name.rsplit("\\", 1)[-1],
-        unc_path=header.file_name,
+        unc_path=NOT_APPLICABLE,
```

Comment:
Is this an expected change? We are replacing the path with `N/A`. This seems like something that should be in its own PR.

Reply:
Thanks for catching up on this